frontend/views/containers/dashboard/GroupMembersTooltipPending.vue

```diff
@@ -17,17 +17,25 @@ export default ({
     Tooltip
   },
   props: {
-    contractID: String
+    contractID: String,
+    data: Object
+  },
+  methods: {
+    getDisplayName (memberID) {
+      const profile = this.globalProfile(memberID)
+      return profile?.displayName || profile?.username || memberID
+    }
   },
   computed: {
     ...mapGetters([
       'ourIdentityContractId',
-      'groupMembersPending'
+      'groupMembersPending',
+      'globalProfile'
     ]),
     tooltipText () {
-      const invitedBy = (this.groupMembersPending[this.username] || {}).invitedBy
+      const invitedBy = this.getDisplayName(this.data?.invitedBy)
 
-      return this.contractID === this.ourIdentityContractId
+      return this.ourIdentityContractId === this.data?.invitedBy
```

Comment:
The existing logic here is apparently based on the legacy logic, causing the bug. So needs to be updated accordingly.